fuzz/subgraph/src/model.rs

```diff
@@ -127,6 +127,7 @@ impl User {
     }
 }
 
+#[allow(static_mut_refs)]
```

Comment:
Why not use `OnceLock` instead? This could also removes the need for unsafe code